configure.ac

```diff
@@ -3403,6 +3403,8 @@ AC_DEFINE_UNQUOTED(DLEXT_MAXLEN, `expr $len + 1`)
 test ".$DLEXT"  = "." || AC_DEFINE_UNQUOTED(DLEXT,  ".$DLEXT")
 AC_SUBST(DLEXT)
 
+AC_DEFINE_UNQUOTED(SOEXT, ".$SOEXT")
```

Comment:
Including a dot, only in config.h?

Reply:
I'm not sure what you mean. This is similar to DLEXT, which includes a dot at the beginning.

Reply:
Sorry, indeed.